specs/derivation.md

```diff
@@ -473,37 +473,43 @@ updated, such that the batch-sender authentication is always accurate to the exa
+  address loaded from the system config matching the L1 block of the data.
 
-Each data-transaction is versioned and contains a series of [channel frames][g-channel-frame] to be read by the
-Frame Queue, see [Batch Submission Wire Format][wire-format].
+Each batcher transaction is versioned and contains a series of [channel frames][g-channel-frame] to
+be read by the Frame Queue, see [Batch Submission Wire Format][wire-format]. Each batcher
+transaction in the block is processed by passing its calldata on to the next phase.
 
 #### Eclipse: Blob Retrieval
 
-With the Eclipse upgrade the retrieval stage is extended to support an additional DA source: [EIP-4844] blobs.
-Upon each traversed transaction in the batch inbox:
+With the Eclipse upgrade the retrieval stage is extended to support an additional DA source:
+[EIP-4844] blobs. With this extension, the above batcher transaction processing is modified to
+consider only batcher transactions with transaction-type != `0x03` (`BLOB_TX_TYPE`). Only after
+fully processing these non-blob transaction do we then consider blob transactions, each of which is
+processed as follows:
 
-- The receiver must be the configured batcher inbox address.
-- The sender must match the batcher address loaded from the system config matching the L1 block of the data.
-- Calldata is utilized, even if contained in a blob tx, and passed on to the next stage before any blob-data is.
-- If the transaction-type is `0x03` (`BLOB_TX_TYPE`): retrieve the blobs that match the `blob_versioned_hashes`.
-  - Blobs may be retrieved from different sources. Retrieval from a local beacon-node,
-    through the `/eth/v1/beacon/blob_sidecars/` endpoint, with `indices` filter to skip unrelated blobs, is recommended.
-  - The rollup node SHOULD cryptographically verify if the retrieved blobs match the versioned hashes.
+- For each blob hash in `blob_versioned_hashes`, retrieve the blob that matches it. A blob may be
+    retrieved from any of a number different sources. Retrieval from a local beacon-node, through
+    the `/eth/v1/beacon/blob_sidecars/` endpoint, with `indices` filter to skip unrelated blobs, is
+    recommended.  For each retrieved blob:
+  - The blob SHOULD (MUST, if the source is untrusted) be cryptographically verified against its
+    versioned hash.
+  - If the blob has a [valid encoding][#blob-encoding], decode it into its continuous byte-string
+    and pass that on to the next phase.
 
-On L1, the blob data is represented as a polynomial of points, each in range `[0, BLS_MODULUS)`,
-just below the capacity of a `uint256`: approximately `254.857` bits of data.
-Before proceeding with processing, the data is transformed to turn it into a continuous byte string.
-
-TODO: blob encoding format. Version + length-prefix + maximize utilization of the data.
+Note that calldata from blob transactions, should it be present, is ignored.
```

Comment:
Can we add in the reasoning behind this?
1. Simplicity in building transactions
2. The derivation pipeline could be extended in the future to include batch metadata in the calldata to be used in future features such as leader election

Reply:
how about just:

```
Note that calldata from blob transactions, should it be present, is ignored, and may be used in the
future for batch metadata or other purposes.
```

Reply:
That is great!